### PR TITLE
Modifications necessary to synchronize with Silvio

### PR DESCRIPTION
**Pull request by swang373**
These were the modifications that allowed me to run the full analysis workflow on lxplus.

### CODE REVIEW

**Comment by swang373** [comment]
I didn't realize I had modified Ratio_C.d, if that's an issue, let me know.

@silviodonato,
I changed the paths in the macros such as SavePlot.C, etc. Do you want me to revert those paths?

**Comment by silviodonato** [comment]
no, I'm ok your current configuration

File: interface/DrawFunctions.C [hunk 1]
```diff
@@ -11,7 +11,7 @@ TH2F* NewOverQCD = 0;
 
 bool Under(float phi, float eta) {
   if (ECALmap == 0) {
-    ECALmap = TFile::Open("/scratch/sdonato/VHbbRun2/V21/CMSSW_7_1_5/src/Xbb/python/plot.root", "r");
+    ECALmap = TFile::Open("/afs/cern.ch/work/s/swang373/private/sync_V21/src/Xbb/python/plot.root", "r");
```
Comment (perrozzi) [inline interface/DrawFunctions.C]: could you use a relative path to let it work for everybody?

File: python/ZvvHbb13TeVmacros/SavePlot.C [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
  TFile* nf = new TFile("newfile.root","recreate");
- TFile *_file0 = TFile::Open("../env/ZvvHighPt_V21_MET.root");
+ TFile *_file0 = TFile::Open("/afs/cern.ch/work/s/swang373/public/V21_MVAout/ZvvHighPt_V21_MET.root");
```
Comment (perrozzi) [inline python/ZvvHbb13TeVmacros/SavePlot.C]: same as above, you and @silviodonato should probably agree on a relative path convention whenever needed

File: python/myutils/Ratio_C.d [hunk 3]
```diff
@@ -1,136 +1,42 @@
```
Comment (perrozzi) [inline python/myutils/Ratio_C.d]: do we really need to keep this file in the repository?!?

**Comment by perrozzi** [comment]
apart from the comments inline in the diff, looks ok to me

File: python/myutils/Ratio_C.d (hunk 3 above)
Comment (swang373) [inline python/myutils/Ratio_C.d]: I doubt it... If no one is opposed, I'll update this pull request with it removed.

File: interface/DrawFunctions.C (hunk 1 above)
Comment (swang373) [inline interface/DrawFunctions.C]: The root file it refers to is generated by getPeaks2.py in the ZvvHbb13TeVMacros directory, so as long as the macro outputs it to the same directory within Xbb, a relative path should work.

File: python/ZvvHbb13TeVmacros/SavePlot.C (hunk 2 above)
Comment (swang373) [inline python/ZvvHbb13TeVmacros/SavePlot.C]: Ah, this one should be reverted. The samples should reside within env/ but I copied them over from Silvio to my AFS public directory.

File: interface/DrawFunctions.C (hunk 1 above)
Comment (perrozzi) [inline interface/DrawFunctions.C]: yes please, in order to avoid commit noise

**Pull request closed without merging**